### PR TITLE
Refactor/remove second editor code

### PR DESCRIPTION
**Pull request by chic-geek**
- Remove the second editor.
- Remove the label.

Closes issue #8 
Closes issue #7

### CODE REVIEW

File: src/components/RichTextEditor/component.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 import Html from 'slate-html-serializer';
 
-import { Value } from 'slate';
 import { IS_BOLD_HOTKEY, IS_ITALIC_HOTKEY, IS_UNDERLINED_HOTKEY } from './constants';
 import { onLinkPaste, renderInline } from './link';
 import { renderMark, renderBlock, rules } from './utils';
 import initialValue from './value.json';
 
+import EditorToolbar from './components/EditorToolbar';
 import EditorLabel from './components/EditorLabel';
```
Comment (dianabit) [inline src/components/RichTextEditor/component.jsx:15]: As part of fixing the conflicts, I kept this line, but I couldn't see that you deleted the `EditorLabel` component. If we don't need it anymore, we should remove it from here too.

Comment (dianabit) [reply src/components/RichTextEditor/component.jsx:15]: And obviously, we also need to remove it from where it is actually rendered below.

**Comment by chic-geek** [comment]
> Looks good. Does this work well with Casebook? The reason why the second editor was added in the first place was that it would not work with both inputs on the same page (summary field and next steps) but lots of stuff have changed since that was tested.

Hmm, I haven't tested this yet. Aren't the instances of an editor isolated by their very nature?

**Comment by dianabit** [comment]
> Hmm, I haven't tested this yet. Aren't the instances of an editor isolated by their very nature?
I am not sure. I suggest testing this in the RTE app by adding another editor and see if there are any evident issues. If it works well, probably everything will be fine in Casebook as well.

**Pull request merged**